cdap-watchdog/src/main/java/io/cdap/cdap/logging/guice/LocalLogAppenderModule.java

```diff
@@ -28,6 +34,16 @@ public class LocalLogAppenderModule extends AbstractModule {
 
   @Override
   protected void configure() {
-    bind(LogAppender.class).to(LocalLogAppender.class).in(Scopes.SINGLETON);
+    bind(LocalLogAppender.class).in(Scopes.SINGLETON);
+    bind(ExtensionLogAppender.class).in(Scopes.SINGLETON);
+  }
+
+  @Provides
+  @Singleton
+  @SuppressWarnings("unused")
+  protected LogAppender provideCompositeLogAppender(LocalLogAppender localLogAppender,
```

Comment:
Why is this protected? Can other classes extend LocalLogAppenderModule and use this function?

Reply:
Well, I would rename the module/function as it's definitely going to do something else than it's current name / javadoc. 
To @vsethi09 question it's providing a binding for `LogAppender` with `@Provides`, so it's going to be called by Guice.